src/lib/hooks/useGetCurrentVoteList.tsx

```diff
@@ -5,9 +5,10 @@ import { VoteResultData } from '../../types/vote';
 import { picmeGetFetcher } from '../axios';
 
 const getKey = (cursorIdx: number, voteListData: AxiosResponse<VoteResultData>) => {
+  console.log(voteListData);
```

Comment:
```suggestion
const { data : voteResultData } = voteListData;
```
로 받아서 사용하는게 더 직관적일것같아요!

Reply:
요건,,,,, 안되는걸루,,,,ㅎㅎ,,,,,,,,, SWRInfinite 어렵네요잉~~ㅠㅅㅠ